includes/class-frontend-render-form.php

```diff
@@ -793,9 +793,13 @@ public static function prepare_meta_fields( $meta_vars ) {
                     $data           = [];
                     $map_field_data = $value_name;
 
-                    if ( !empty( $map_field_data ) ) {
-                        list( $data['address'], $data['lat'], $data['lng'] ) = explode( ' || ', $map_field_data );
-                        $meta_key_value[$value['name']]                      = $data;
+                    if ( ! empty( $map_field_data ) ) {
+                        if (stripos( $map_field_data, '||' ) !== false ){
+                            list( $data['address'], $data['lat'], $data['lng'] ) = explode( ' || ', $map_field_data );
+                            $meta_key_value[$value['name']]  = $data;
+                        }else{
```

Comment:
Please fix the space here. Also, do we know it'll be JSON data for sure? Maybe someone doesn't use ACF, what will happen then?

Reply:
we are storing 3 keys lat,lng & address. acf is storing more fileds alongside with these. so if we store all of these keys it will consistent with acf & without acf. we are sure this data is json format. besides we are checking here for old format.